storage/monitoring.go

```diff
@@ -0,0 +1,17 @@
+package storage
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/atomic"
+)
+
+var log = logger.GetOrCreate("storage")
+
+var cummulatedSizeInBytes atomic.Counter
+
+// Question for review: keep this or remove it (helps us to compute planned memory at runtime)?
+func MonitorNewCache(tag string, sizeInBytes uint64) {
```

Comment:
:+1: It is useful (at least for the time being). Might move the functionality directly in the NewCache function.

Reply:
Moved.